### ISSUE TITLE
Keras - Layers

### ISSUE THREAD

**Issue by junxnone** [junxnone/tech-io#702]
# Layers

- [ ] 核心层 Core 
  - [ ] Dense
  - [ ] Activation
  - [ ] Dropout
  - [ ] Flatten
  - [ ] Input
  - [ ] Reshape
  - [ ] Permute
  - [ ] RepeatVector
  - [ ] Lambda
  - [ ] ActivityRegularization
  - [ ] Masking
  - [ ] SpatialDropout1D
  - [ ] SpatialDropout2D
  - [ ] SpatialDropout3D
- [ ] 卷积层 Convolutional
  - [ ] Conv1D
  - [ ] Conv2D
  - [ ] SeparableConv1D
  - [ ] SeparableConv2D
  - [ ] Conv2DTranspose
  - [ ] Conv3D
  - [ ] Cropping1D
  - [ ] Cropping2D
  - [ ] Cropping3D
  - [ ] UpSampling1D
  - [ ] UpSampling2D
  - [ ] UpSampling3D
  - [ ] ZeroPadding1D
  - [ ] ZeroPadding2D
  - [ ] ZeroPadding3D
- [ ] 池化层 Pooling
  - [ ] MaxPooling1D
  - [ ] MaxPooling2D
  - [ ] MaxPooling3D
  - [ ] AveragePooling1D
  - [ ] AveragePooling2D
  - [ ] AveragePooling3D
  - [ ] GlobalMaxPooling1D
  - [ ] GlobalAveragePooling1D
  - [ ] GlobalMaxPooling2D
  - [ ] GlobalAveragePooling2D
  - [ ] GlobalMaxPooling3D
  - [ ] GlobalAveragePooling3D
- [ ] 局部连接层 Locally-connected
  - [ ] LocallyConnected1D
  - [ ] LocallyConnected2D
- [ ] 循环层 Recurrent
  - [ ] RNN
  - [ ] SimpleRNN
  - [ ] GRU
  - [ ] LSTM
  - [ ] ConvLSTM2D
  - [ ] SimpleRNNCell
  - [ ] GRUCell
  - [ ] LSTMCell
  - [ ] StackedRNNCells
  - [ ] CuDNNGRU
  - [ ] CuDNNLSTM
- [ ] 嵌入层 Embedding
  - [ ] Embedding
- [ ] 融合层 Merge
  - [ ] Add
  - [ ] Subtract
  - [ ] Multiply
  - [ ] Average
  - [ ] Maximum
  - [ ] Concatenate
  - [ ] Dot
- [ ] 高级激活层 Advanced Activations
  - [ ] LeakyReLU
  - [ ] PReLU
  - [ ] ELU
  - [ ] ThresholdedReLU
  - [ ] Softmax
  - [ ] ReLU
- [ ] 标准化层 Normalization
  - [ ] BatchNormalization
- [ ] 噪声层 Noise
  - [ ] GaussianNoise
  - [ ] GaussianDropout
  - [ ] AlphaDropout
- [ ] 自定义Keras层

**Comment by junxnone** [comment]
#9
